dotnet/src/IntegrationTests/Connectors/Memory/AzureCosmosDBNoSQL/AzureCosmosDBNoSQLVectorStoreTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Azure.Cosmos;
+using Microsoft.SemanticKernel.Connectors.AzureCosmosDBNoSQL;
+using Xunit;
+
+namespace SemanticKernel.IntegrationTests.Connectors.Memory.AzureCosmosDBNoSQL;
+
+/// <summary>
+/// Unit tests for <see cref="AzureCosmosDBNoSQLVectorStore"/>.
+/// </summary>
+[Collection("AzureCosmosDBNoSQLVectorStoreCollection")]
+public sealed class AzureCosmosDBNoSQLVectorStoreTests(AzureCosmosDBNoSQLVectorStoreFixture fixture)
+{
+    private const string? SkipReason = "Azure CosmosDB NoSQL cluster is required";
+
+    [Fact(Skip = SkipReason)]
+    public async Task ItCanGetAListOfExistingCollectionNamesAsync()
+    {
+        // Arrange
+        var sut = new AzureCosmosDBNoSQLVectorStore(fixture.Database!);
+
+        await fixture.Database!.CreateContainerIfNotExistsAsync(new ContainerProperties("sk-test-hotels", "/id"));
+        await fixture.Database!.CreateContainerIfNotExistsAsync(new ContainerProperties("sk-test-contacts", "/id"));
```

Comment:
Just a suggestion, but to avoid accidental clashes I started using names that reflect the test, e.g. in this case something like `list-names-1` and `list-names-1`.